internal/provider/resource/identity_oidc_auth.go

```diff
@@ -98,6 +99,16 @@ func (r *IdentityOidcAuthResource) Schema(_ context.Context, _ resource.SchemaRe
 					pkg.CommaSpaceMapModifier{},
 				},
 			},
+
+			"claim_metadata_mapping": schema.MapAttribute{
+				Description: "The attributes that should be present in the permission metadata from the JWT.",
+				Optional:    true,
+				Computed:    true,
+				ElementType: types.StringType,
+				PlanModifiers: []planmodifier.Map{
+					mapplanmodifier.UseStateForUnknown(),
+				},
+			},
```

Comment:
it's not just about checking whether its present right? it's also about actually mapping the oidc JWT claims to the metadata? 

At first glance, it wasn't clear to me what this field is really for